frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -71,14 +76,19 @@
         <lf-button type="secondary-ghost" @click="close">
           Cancel
         </lf-button>
-        <lf-button
-          type="primary"
-          :loading="sending"
-          :disabled="$v.$invalid || sending"
-          @click="updateWorkExperience()"
+        <lf-tooltip
+          :disabled="!(hasSameOrgDetails && !isEdit)"
```

Comment:
This logic (when it has the same org details) should be applied wether or not the user is in edit or add mode. Could you remove `!isEdit`?